Makefile

```diff
@@ -6,6 +6,9 @@ build:
 start:
 	pipenv run python run.py
 
-test:
-	pipenv check --style ./secure_message ./tests
-	pipenv run pytest
+lint:
+	pipenv run flake8 ./secure_message ./tests
+	pipenv check ./secure_message ./tests
+
+test: lint
+    pipenv run run_tests.py
```

Comment:
I think the makefile needs to be all tabbed, otherwise we get that missing separator error in Travis

Reply:
Good spot! PyCharm has converted my tab to spaces 😠